task-launcher/src/tasks/shared/helpers/fetchAndParseCorpus.js

```diff
@@ -66,34 +64,26 @@ const transformCSV = (csvInput, numOfPracticeTrials, sequentialStimulus) => {
       newRow.distractors = newRow.distractors.map((choice) => camelize(choice));
     }
 
-    totalTrials += 1;
-
-    if (row.notes === 'practice') {
-      totalPractice += 1;
-    }
-
     let currentTrialType = newRow.trialType;
-
-    // console.log('currentTrialType:', currentTrialType)
-    // console.log('currTrialTypeBlock:', currTrialTypeBlock)
-
     if (currentTrialType !== currTrialTypeBlock) {
       currTrialTypeBlock = currentTrialType;
       currPracticeAmount = 0;
     }
 
-    // Only push in the specified amount of practice trials
-    if (newRow.notes !== 'practice') {
```

Comment:
refactored this code block, preserving the original desire for stimulusData to contain all trials